config/default/LtiProviderFieldsMapper.conf.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use oat\tao\model\oauth\DataStore;
+use oat\taoLti\models\classes\LtiProvider\ConfigurableLtiProviderRepository;
+use oat\taoLti\models\classes\LtiProvider\LtiProviderFieldsMapper;
+use oat\taoLti\models\classes\LtiProvider\RdfLtiProviderRepository;
+
+return new LtiProviderFieldsMapper(
```

Comment:
Do you think it is needed to have this configured externally? Seems internal needs only. Wdyt?